apps/site/lib/site_web/controllers/schedule/trip_view_controller.ex

```diff
@@ -34,4 +38,8 @@ defmodule SiteWeb.ScheduleController.TripViewController do
   defp zone_map(conn, _), do: conn
 
   defp tab_name(conn, _), do: assign(conn, :tab, "trip-view")
+
+  defp call_plug_with_opts(conn, module, opts) do
```

Comment:
👍 Seems like this could be a broadly useful helper, maybe put this in `SiteWeb.ControllerHelpers`?